ensemble_docker_pipelines/README.md

```diff
@@ -0,0 +1,130 @@
+# Docker pipelines to prepare other callers output for Ensemble mode
+
+
```

Comment:
You can mention that this is an adaptation of SomaticSeq's scripts. I think, in the longer term, a Nextflow script kind of solution would be good instead of these independent scripts.